synth.py

```diff
@@ -7,12 +7,16 @@
 s.copy(templates, excludes=[
   '.eslintignore',
   '.eslintrc.yml',
+  '.github/release-please.yml',
+  '.github/workflows/',
+  '.github/publish.yml',
+  '.kokoro/*.*',
+  '.kokoro/**/*.*',
```

Comment:
did some testing, this can just be the glob `.kokoro/**`.

Reply:
Great catch!  I can't believe I missed that.